### PR TITLE
refactor(styles): convert css files to js

### PR DESCRIPTION
**Pull request by sidvishnoi**
This PR converts all `fetchAsset(file.css)` and `import(text!file.css)` to regular JS imports. 
The benefit being, cleaner code and less async-await, and slightly smaller bundle size. Worth doing?

<details>
<summary>Size changes</summary>

| File             |  After | Before | Diff      |
| ---------------- | -----: | -----: | --------: |
| respec-w3c.js    | 305747 | 308252 | -2.5Kb 🟢 |
| respec-w3c.js.br |  84910 |  84815 | +0.1Kb 🔴 |
| respec-w3c.js.gz |  99233 |  98219 | +1.0Kb 🔴 |

</details>

### CODE REVIEW

**Review by marcoscaceres — COMMENTED** [review]
If we are doing this, then we might as well just get rid of the *.css.js files and fold them into the modules. Then we can do proper "CSS in JS" stuff.

**Comment by saschanaz** [comment]
Yeah, I agree that it would be better, assuming that every CSS file is small enough and has no visible performance degradation from this.

**Comment by sidvishnoi** [comment]
Wouldn't regular css-in-js require a build step? `.css.js` files don't change our development workflow (directly using src/ without building).

File: src/core/dfn-index.js [hunk 1]
```diff
@@ -46,14 +46,14 @@ const CODE_TYPES = new Set([
  * @typedef {{ term: string, type: string, linkFor: string, elem: HTMLAnchorElement }} Entry
  */
 
-export async function run() {
+export function run() {
```
Comment (sidvishnoi) [inline src/core/dfn-index.js:49]: Should I keep the async here for consistency? Or, not add async where not needed?

Comment (saschanaz) [reply src/core/dfn-index.js:49]: I would just remove it.

**Comment by saschanaz** [comment]
> Wouldn't regular css-in-js require a build step? `.css.js` files don't change our development workflow (directly using src/ without building).

Well, I mean, why import it when we can just put it right in the modules? That shouldn't require any build step.

**Comment by sidvishnoi** [comment]
> why import it when we can just put it right in the modules

I think they're large enough to look bad in plugin files:
```bash
# ls assets/* | xargs wc -l | sort -n
    7 assets/algorithms.css
   36 assets/dfn-index.css
   43 assets/examples.css
   45 assets/datatype.css
   49 assets/var.css
   54 assets/issues-notes.css
   94 assets/highlight.css
  111 assets/dfn-panel.css
  127 assets/caniuse.css
  136 assets/mdn-annotation.css
  140 assets/webidl.css
  285 assets/respec.css
  356 assets/ui.css
```

**Review by saschanaz — APPROVED** [review]
Ooops, 100+ line files are definitely too big.

Okay, I support this, assuming that the minifier works as expected.

**Comment by sidvishnoi** [comment]
Minification works as expected:
| Non-minified (now) | Minified (now) | Minified (before) |
| -- | -- | -- |
| image | image | image |

**Comment by saschanaz** [comment]
Cool! Interesting that it keeps newlines after each rule 👀

**Comment by sidvishnoi** [comment]
The newlines were there before, but as `\n` in `""`, now they're in backticks.

**Review by marcoscaceres — APPROVED** [review]
Ok, let's start here.

**Pull request merged**